dist/js/axs_testing.js

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
```

Comment:
Please revert the change to the compiled file - we only update this for releases.